package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "hex-rgb",
-	"version": "3.0.0",
+	"version": "3.1.0",
```

Comment:
Don't bump the version in other people's project. It's up to the maintainers to decide which version and when to bump.